front/src/modules/favorites/components/Favorites.tsx

```diff
@@ -14,46 +20,90 @@ const StyledContainer = styled.div`
 `;
 
 export const Favorites = () => {
-  const { data } = useGetFavoritesQuery();
-  const favorites = data?.findFavorites;
+  const { updateFavoritesOrder } = useFavorites();
+
+  const [favorites, setFavorites] = useState<
+    GetFavoritesQuery['findFavorites']
```

Comment:
I think this is the right pattern : introducing a state and setting it on query completion. 

We could argue to use a recoil state (an atom) instead of a local state:
- this would make the favorites Api easier to use, as you read and write to a shared state, you don't need to pass the favorites to the favorites setFavorites function (the favorites function knows where to find it)

Could you:
- create an atom (type Pick<Favorites, 'id' | ...>, see my comment below
- move the content of handleReorder to useFavoritesHooks, this hook will update the favorites atom + perform the call to the backend

Reply:
Thanks for this. It will be great with recoil state. I will add this